src/wiki/development/build-instructions.md

```diff
@@ -214,10 +214,10 @@ We recommend using a build workflow based on MSYS2, as it's the easiest way to g
 
 1. Open the right **MSYS2 MinGW** shell and clone PolyMC by doing `git clone --recursive https://github.com/PolyMC/PolyMC.git`, and change directory to the folder you cloned to.
 2. Now we can prepare the build itself: Run `cmake -Bbuild -DCMAKE_INSTALL_PREFIX=install`. These options will copy the final build to `C:\msys64\home\<your username>\PolyMC\install` after the build.
-3. If you want PolyMC to store its data in `%APPDATA%`, append `-DLauncher_PORTABLE=OFF` to the previous command.
-4. Now you need to run the build itself: Run `cmake --build build -jX`, where *X* is the number of cores your CPU has.
-5. Now, wait for it to compile. This could take some time, so hopefully it compiles properly.
-6. Run the command `cmake --install build`, and it should install PolyMC to whatever the `-DCMAKE_INSTALL_PREFIX` was.
+3. Now you need to run the build itself: Run `cmake --build build -jX`, where *X* is the number of cores your CPU has.
+4. Now, wait for it to compile. This could take some time, so hopefully it compiles properly.
+5. Run the command `cmake --install build`, and it should install PolyMC to whatever the `-DCMAKE_INSTALL_PREFIX` was.
+6. If you don't want PolyMC to store its data in `%APPDATA%`, run `cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }} --component portable` after the install process
```

Comment:
copypaste moment from GHA

Reply:
${{ env.XXXX }} are gha remove those and replace with the actual dirs referenced above

Reply:
oops